src/context.d.ts

```diff
@@ -18,6 +18,9 @@ export interface Context {
   /** The current projection, if any. */
   projection?: GeoStreamWrapper;
 
+  /** A function to draw GeoJSON with the current projection, if any, otherwise with the x and y scales. */
+  path?: () => GeoPath;
```

Comment:
```suggestion
  /** A function to draw GeoJSON with the current projection, if any, otherwise with the x and y scales. */
  path: () => GeoPath;
```